docs/man/git-lfs-config.5.ronn

```diff
@@ -190,6 +184,22 @@ lfs option can be scoped inside the configuration for a remote.
   If set to "basic" then credentials will be requested before making batch
   requests to this url, otherwise a public request will initially be attempted.
 
+* `lfs.<url>.skiplocksverify`
+
+  Determines whether locks are checked before Git pushes. This prevents you from
+  pushing changes to files that other users have locked. The Git LFS pre-push
+  hook varies its behavior based on the value of this config key.
+
+  * `null` - In the absence of a value, Git LFS will attempt the call, and warn
```

Comment:
Actually, I think the values are correct. I think I just forgot to switch the true/false values when I changed the name to `skipverifylocks`. Now that it's `locksverify`, I think it's okay now :)